lighthouse-core/config/default.json

```diff
@@ -3,6 +3,7 @@
     "passName": "defaultPass",
     "recordNetwork": true,
     "recordTrace": true,
+    "pauseAfterTraceMs": 500,
```

Comment:
yah. i actually wrote that first. :) and then revised so it looks more similar to `recordTrace` since they're buddies

but yeah let's go with pauseBeforeTraceEnd since we both went there.